lib/login/controller.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:firebase/firebase.dart' as firebase;
+import 'package:dashboard/login/view.dart' as login_view;
+import 'package:dashboard/firebase.dart' as fb;
+import 'package:dashboard/logger.dart';
+
+Logger logger = Logger('loginModal/controller.dart');
```

Comment:
consider just `login/` instead of `loginModal/` (ie. the file path rather than object name)?